docs/versioned_docs/version-zenith/developer-guide/python/820256-module-structure.mdx

```diff
@@ -135,27 +141,30 @@ from .lint import Lint  # in src/main/lint.py
 ## Project name
 
 The default project name in `pyproject.toml` is also `main`:
+
 ```toml
 [project]
 name = "main"
 ```
 
- This is because most package managers use the name of the project to find the package to build, under `./src`. So, if you change the name of the project, you also need to make sure your chosen tool can find the package.
+This is because most package managers use the name of the project to find the package to build, under `./src`. So, if you change the name of the project, you also need to make sure your chosen tool can find the package.
+
+Here is an example of using a different project name with Poetry (truncated for brevity):
 
- For example, with Poetry, using a different project name (truncated for brevity):
 ```toml
 [tool.poetry]
 name = "awesome-module"
 packages = [{ include = "main.py", from = "src" }]
 ```
 
-## src layout
+## Default project layout
```

Comment:
"src layout" is a thing in Python (https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/), that's why I used it. The emphasis was intentional.

It used to be a common pitfall to not put sources in `src`, so there's lots writing on the web about using a "src layout", meaning, put all your python files in a `src` directory, that is **not your working directory**.

Now it's a convention that many tools default to.

Reply:
Didn't realize this convention, reverted